### PR TITLE
Resolves #78, resolves #79 Translate Test Strings and Remove $(PRODUCT_NAME)

### LINKED ISSUE #43: Unnecessary strings in XLIFF
Please take a look at the crowdin options, there are many arguments and placeholders for translation, as you can see here (should not be available for translation):
https://github.com/openfoodfacts/openfoodfacts-ios/blob/master/xliff/en.xliff#L541

Thanks!

### LINKED ISSUE #78: $(PRODUCT_NAME) does not need to be translatable
$(PRODUCT_NAME) does not need to be translatable

### LINKED ISSUE #79: String confusing translators
image

Those strings are confusing translators. They should either be clarified or made untranslatable.

### PR DESCRIPTION
**Pull request by snuff4**
## PR Description

Translated the Test strings

Type of Changes 

- [x] Fixes Issue #78  #79 

## Checklist
 
Make sure you've done all the following (_Put an `x` in the boxes that apply._)
 
 - [x] If you have multiple commits please combine them into one commit by [squashing](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) them. 
 - [ ] Code is well documented
 - [x] Included unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [x] Code passes Travis builds in your branch

Edit 1: Does not resolves issue #43 I don't know if those strings should be considered unnecessary or not.

### CODE REVIEW

File: xliff/en.xliff [hunk 1]
```diff
@@ -700,84 +696,84 @@ Make sure the barcode is well lit.</target>
...
-        <target xml:lang="en-GB" state="needs-translation">user.alert.password-missing</target>
+        <target xml:lang="en-GB" state="needs-translation">Password is required</target>
       </trans-unit>
       <trans-unit id="user.alert.username-missing">
-        <source>user.alert.username-missing</source>
+        <source>Username is required</source>
         <note>No comment provided by engineer.</note>
-        <target xml:lang="en-GB" state="needs-translation">user.alert.username-missing</target>
+        <target xml:lang="en-GB" state="needs-translation">Username is required</target>
       </trans-unit>
```
Comment (snuff4) [inline xliff/en.xliff]: These changes may lead to repetition of strings in Crowdin translations. I need to find a better solution, before we can merge this.

**Comment by teolemon** [comment]
@roger-tan Good for merging ?

**Comment by snuff4** [comment]
@teolemon There may be repetition of strings in Crowdin translations. Either we can merge it to check it, because this PR will take care of the weird strings that were Test Cases.

**Comment by teolemon** [comment]
Merging, since @roger-tan approved.

**Pull request merged**